packages/npm-vue/README.md

```diff
@@ -0,0 +1,52 @@
+Reatom integration for Vue Composition API.
+
+## Installation
+
+```sh
+npm i @reatom/npm-vue
+```
+
+## API
+
+### `reatomRef`
+
+A function that turns a Reatom atom into a Vue ref which is updated on target atom changes. A returned pseudo-ref is mutable if a target atom is mutable itself.
+
+Because all Reatom APIs require `ctx` to be available, you must either `provide` it with `reatomCtxKey` or pass it explicitly as a second argument to `reatomRef`.
+
+### `reatomCtxKey`
+
+A symbol to provide the integration with a `ctx` using Vue Dependency Injection API.
+
+## Example
+
+Setup `ctx` somewhere in the app root:
+
+```ts
+import { createCtx } from '@reatom/core'
```

Comment:
For typical Vue-way it would be better to create [App-plugin](https://vuejs.org/guide/reusability/plugins.html#provide-inject-with-plugins) that provides the context (as in [Vuex](https://vuex.vuejs.org/guide/#the-simplest-store) / [Pinia](https://pinia.vuejs.org/getting-started.html#Installation) etc).
```ts
// index.ts
import { createApp } from 'vue'
import { reatomCtxKey} from '@reatom/npm-vue'
import App from './App.vue'

const app = createApp(App)
app.use(createReatomVue())
```